### ISSUE TITLE
Color Pallete for Website

### ISSUE THREAD

**Issue by kmrakash** [sajag1999/Website-Blocker-Project#48]
Please help me in this issue
create color vairable in main.css file
it must include:

- Primary Color
- Secondary Color
- Accent Color
- Text Color
- Background Color

you can include more as per requirement

**Comment by Aakansha99** [comment]
yes, i can help in color pallete..mentors kindly assign me this issue

**Comment by Swarnimashukla** [comment]
Would you guys like to collaborate in this issue ? @Aakansha99  @kmrakash  then I will assign this issue to both of you .

**Comment by Aakansha99** [comment]
Yes...I think this will be better
